### ISSUE TITLE
Caussed by: android.system.ErrnoException - statvfs failed: ENOENT (No such file or directory)

### ISSUE THREAD

**Issue by puzanshrestha** [fingerprintjs/fingerprintjs-android#90]
Crashes on Samsung devices
Samsung
Galaxy S23 Ultra
Android 13

```
libcore.io.Linux.statvfs (Linux.java)
libcore.io.ForwardingOs.statvfs (ForwardingOs.java:852)
libcore.io.BlockGuardOs.statvfs (BlockGuardOs.java:428)
libcore.io.ForwardingOs.statvfs (ForwardingOs.java:852)
android.system.Os.statvfs (Os.java:909)
android.os.StatFs.doStat (StatFs.java:51)
android.os.StatFs.<init> (StatFs.java:43)
com.fingerprintjs.android.fingerprint.FingerprinterFactory.createMemoryInfoProvider (FingerprinterFactory.kt:145)
com.fingerprintjs.android.fingerprint.FingerprinterFactory.createHardwareSignalGroupProvider (FingerprinterFactory.kt:92)
com.fingerprintjs.android.fingerprint.FingerprinterFactory.initializeFingerprinter (FingerprinterFactory.kt:79)
com.fingerprintjs.android.fingerprint.FingerprinterFactory.getInstance (FingerprinterFactory.kt:63)
```

**Comment by Sergey-Makarov** [comment]
Hey! Thanks for your report!

First of all, could you please tell which version of the library you are using?
Judging by the stacktrace provided, the version is something below `1.2.1`, which is quite old.
So I would suggest you to update to the latest version, which is `2.0.1` at the moment, and check whether the issue still remains.
We had a PR that was merged into `1.2.1` that can potentially address exactly this issue, so it's definitely worth trying :)

**Comment by puzanshrestha** [comment]
> Hey! Thanks for your report!
> 
> First of all, could you please tell which version of the library you are using? Judging by the stacktrace provided, the version is something below `1.2.1`, which is quite old. So I would suggest you to update to the latest version, which is `2.0.1` at the moment, and check whether the issue still remains. We had a PR that was merged into `1.2.1` that can potentially address exactly this issue, so it's definitely worth trying :)

Thanks @Sergey-Makarov. Yes I have been using older version 1.1, will update to latest version.
